### ISSUE TITLE
Aanpassingen voor ingelogde gebruikers (niet admin)

### ISSUE THREAD

**Issue by peterdesmet** [inbo/vespa-db#193]
INSIDE: ingelogde gebruiker met toegang tot gemeente waar nest is, OUTSIDE: ingelogde gebruiker zonder toegang tot gemeente waar nest is

- [ ] `reserved_by`
  - [x] read only voor OUTSIDE
  - [ ] Reserveerknop weghalen, zie #135 
- [ ] Momenteel private, maar mag publiek, en dus ook voor ingelogde gebruikers (zie #192)
- [ ] `wn_admin_notes`: moet private (enkel toegang voor ADMIN), momenteel is dit veld zichtbaar voor gebruikers met toegang to private data

**Comment by steveritz** [comment]
@peterdesmet speelt de personal_data_access token hier nog een rol? Zo dan?
INSIDE: ingelogde gebruiker met toegang tot gemeente waar nest is + personal data access = true
OUTSIDE: ingelogde gebruiker zonder toegang tot gemeente waar nest is

**Comment by peterdesmet** [comment]
Nee, op bovenstaande aanpassingen heeft `personal data access = true` geen invloed. Die setting is enkel van belang voor de 3 observer velden.

**Comment by steveritz** [comment]
@peterdesmet reserved_by enkel voor outside users. Voor de duidelijkheid: alle andere gebruikers kunnen niet zien of een observatie gereserveerd is dan

**Comment by peterdesmet** [comment]
Dat snap ik niet helemaal. Dit is wat gewenst is:

- Status `reserved`: zichtbaar voor alle gebruikers, inclusief publiek
- Zie wie gereserveerd heeft en hoelang: alle ingelogde gebruikers
- Kunnen reserveren (edit rights, i.e. de knop): ingelogde gebruikers binnen een gebied, op voorwaarde dat het nog niet gereserveerd is

**Comment by peterdesmet** [comment]
Het veld `wn_admin_notes` is nog zichtbaar geworden. Het eenvoudigst is om het volledig te verwijderen van de frontend en API (enkel toegang via admin).

**Comment by peterdesmet** [comment]
Het veld `Opmerking validator` (wn_notes) is nog steeds zichtbaar op de front-end voor ingelogde gebruikers.

**Comment by stevegerrits** [comment]
@peterdesmet bedoel je wn_admin_notes of wn_notes? indien wn_notes, moet dit enkel zichtbaar zijn voor admins dan?

**Comment by peterdesmet** [comment]
Excuses voor verwarring, ik bedoel `wn_admin_notes` verwijderen van frontend en API.

**Comment by peterdesmet** [comment]
Closing in favour of #252
